noxfile.py

```diff
@@ -29,6 +29,14 @@ def test(session: nox.Session) -> None:
     session.run("pytest", *session.posargs)
 
 
+@nox.session(python=["3.8"])
```

Comment:
is it assured that the other python version won't have an issue if 3.8 is happy?

Reply:
No, that is not assured. But since we're restricting this for only this `mypy` check, and we only use `mypy` for testing, this should not affect end-users.

The Python API for typing has changed significantly recently, and I thought it was worth focusing on type-checking for one of them.